tests/csm/rest/test_capacity_usage.py

```diff
@@ -173,15 +178,15 @@ def teardown_method(self):
         Teardowm method for deleting s3 account created in setup.
         """
         self.log.info("[START] Teardown Method")
-        if self.restore_pod:
-            self.log.info("Failed deployments : %s", self.failed_pod)
-            for deploy_name in self.failed_pod:
-                self.log.info("[Start]  Restore deleted pods : %s", deploy_name)
-                resp = self.master.create_pod_replicas(num_replica=1, deploy=deploy_name)
+        if self.restore_pod_data:
+            self.log.info("Restore deleted data pods.")
+            for failure_cnt, _ in enumerate(self.restore_list):
```

Comment:
Unnecessary enumerate.

Reply:
I had used in range(len) but that gave codacy issue -> consider using enumerate instead of iterating with range and len (consider-using-enumerate) in the 1st commit, used enumerate

Reply:
Use the list directly
for restore_pod_info in restore_list: